build.sbt

```diff
@@ -165,10 +167,28 @@ lazy val ephemeris = project
     testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-l", "gem.test.Tags.RequiresNetwork"),
     libraryDependencies ++= Seq(
       "org.http4s"    %% "http4s-async-http-client" % http4sVersion,
-      "org.typelevel" %% "mouse"                    % mouseVersion,
+      "org.typelevel" %% "mouse"                    % mouseVersion
       // GspCoreDb.value,
       // GspCoreTestkit.value,
       // Mouse.value,
       // Fs2IO
     )
   )
+
+lazy val util = crossProject(JVMPlatform, JSPlatform)
```

Comment:
I added `Zipper` to this `util` module. Do you think that's fine? Note that it is named `gpp` rather  than`gsp`